src/rpc/misc.cpp

```diff
@@ -1065,20 +1065,22 @@ UniValue getaddresstxids(const UniValue& params, bool fHelp)
     std::vector<std::pair<CAddressIndexKey, CAmount>> addressIndex;
     getAddressesInHeightRange(params, start, end, addresses, addressIndex);
 
-    // This is an ordered set, sorted by height, so result also sorted by height.
-    std::set<std::pair<int, std::string>> txids;
+    // This is an ordered set, sorted by (height,txindex) so result also sorted by height.
+    std::set<std::tuple<int, int, std::string>> txids;
```

Comment:
Update the RPC documentation to say that from zcashd 4.5.0, txids are returned in the order they appear in blocks, which ensures that they are topologically sorted (i.e. parent txids will appear before child txids).

Reply:
keys.txt/D1DB  F2C4  B96F  2DEB  F4C1  6651  0108  112E  7EA8  1F

Reply:
Done, added the documentation